vignettes/estimate_ascertainment.Rmd

```diff
@@ -0,0 +1,294 @@
+
+```{r}
+# get Covid data provided with the package
+data("covid_data")
+
+# filter for the U.K
+df_covid_uk <- filter(covid_data, country == "United Kingdom")
+
+# view the data format
+tail(df_covid_uk)
+```
+
+We then subset the data to focus on just the first few months of the outbreak in the United Kingdom.
+
+```{r}
+# the first few months of the pandemic
+df_covid_uk_subset <- filter(df_covid_uk, date <= "2020-05-31")
+```
+
+Then, we plot case incidence data with following command.
+Further plotting commands are hidden for brevity.
+
+```{r fig.cap="Incidence of cases over time for the ongoing COVID-19 outbreak in the U.K.", class.source = 'fold-hide'}
+ggplot(df_covid_uk_subset) +
+  geom_step(
+    aes(
+      x = date, y = cases
+    ),
+    colour = "steelblue"
+  ) +
+  scale_x_date(
+    date_labels = "%b-%Y"
+  ) +
+  scale_y_continuous(
+    labels = comma
+  ) +
+  labs(
+    x = "Date", y = "Cases"
+  ) +
+  coord_cartesian(
+      expand = FALSE
+  ) +
+  theme_classic() +
+  theme(legend.position = "top")
+```
+
+Then, we plot the death incidence data with following command:
+
+```{r fig.cap="Incidence of deaths over time for the ongoing COVID-19 outbreak in the U.K.", class.source = 'fold-hide'}
+ggplot(df_covid_uk_subset) +
+  geom_step(
+    aes(
+      x = date, y = deaths
+    ),
+    colour = "brown"
+  ) +
+  scale_x_date(
+    date_labels = "%b-%Y"
+  ) +
+  scale_y_continuous(
+    labels = comma
+  ) +
+  labs(
+    x = "Date", y = "Deaths"
+  ) +
+  coord_cartesian(
+      expand = FALSE
+  ) +
+  theme_classic() +
+  theme(legend.position = "top")
+```
+
+### Onset-to-death distribution for Covid-19
+
+We retrieve the appropriate distribution reported in @linton2020 using the `epidist_db()` function from the [_epiparameter_ package](https://epiverse-trace.github.io/epiparameter/).
+
+```{r}
+onset_to_death_covid <- epidist_db(
+  disease = "COVID-19",
+  epi_dist = "onset_to_death",
+  author = "Linton_etal"
+)
+```
+
+### Estimating the proportion of cases that have been ascertained
+
+We use the `estimate_ascertainment()` function within the _cfr_ package to calculate the time-varying CFR for the COVID-19 epidemic in the U.K.
+
+The function includes a `type` argument, which determines whether `estimate_static()` or `estimate_time_varying()` is used to estimate the delay-adjusted severity of the disease.
+
+The ascertainment rate is calculated as the disease severity calculated from the data, divided by the 'known' disease severity; this is expected to be known or assumed from our best knowledge of the pathology of the disease.
+
+This known disease severity is passed to the `severity_baseline` argument in `estimate_ascertainment()`, and forms the denominator in the resulting under-ascertainment calculation.
+
+We assume that the 'true' CFR of Covid-19 is 0.014.
+
+The other arguments are the same as those found in `estimate_time_varying()`. 
+
+```{r }
+df_reporting_static <- estimate_ascertainment(
```

Comment:
Thanks for flagging this - this is related to recent changes to {epiparameter}. Issue #80 has flagged these for edits. Essentially, `epidist_db()` now returns a list rather than an automatically chosen `<epidist>`. I will also replace the parameter access with `epiparameter::get_parameter()`.